src/main/net/polydawn/mdm/MdmArgumentParser.java

```diff
@@ -63,6 +63,12 @@ public void onAttach(Argument arg) {}
 		Subparser parser_status = subparsers
 			.addParser("status")
 			.help("list dependencies managed by mdm, and their current status.");
+		parser_status
+			.addArgument("--name")
+			.help("get the status of a particular dependency by name.  If specified, only that dependency will be included in the report.");
+		parser_status
+			.addArgument("--format")
```

Comment:
Yeah, I hesitated about that for a while.  Then I read up on what git-log does for that, and decided to follow them: git-log has a --format flag with a bunch of preconfigured named options, and then if you want a custom one, the escape valve is to use a value of "format:<string>"... which is a bit redundant, but well, only in a place where terseness is already a lost battle.

I'm too unknowledgable about the state of java string templating systems to take a crack at that, but I'll at least add a comment about that thought on how to expand in the future.